### PR TITLE
feat(posts[routes, controllers]): Delete a specific post

### PR DESCRIPTION
**Pull request by Musigwa-zz**
#### What does this PR do?
It helps deleting a specific post by its ID
#### Description of Task to be completed?
* Allow a user to delete a post
#### How should this be manually tested?
* Clone the repository and run the dependencies installation command `yarn` or `npm install`
* Run `cp sample.env ./.env` to create the `.env` file and modify it to meet your configs
* Run the command to start the server `yarn server` or `npm run server`
* Open your favolite `REST CLIENT TOOL` (usually `postman`)
* Try requesting from the following APIs:
    * `POST /api/v1/posts` to create the post.
    * `DELETE  /api/v1/posts/<post_id>` to delete any of the created posts by its ID.

#### What are the relevant pivotal tracker stories?
#163514316

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -17,7 +17,11 @@ app.use(express.json());
 app.use(express.urlencoded({ extended: true }));
 if (app.get('env') === 'development') app.use(morgan('dev'));
 
+<<<<<<< HEAD
```
Comment (espoirMur) [inline index.js]: @Musigwa, Did you consider fixing conflicts in this file before raising the PR??

**Review by espoirMur — COMMENTED** [review]
@Musigwa, can you consider using the recommended naming conventions for this PR title?

File: index.js (hunk 1 above)
Comment (Musigwa-zz) [reply index.js]: Hello @espoirMur, I think there is are no conflicts in this index.js file. Would you please double check the file to make sure? TIA.

**Comment by Musigwa-zz** [comment]
> @Musigwa, can you consider using the recommended naming conventions for this PR title?



> @Musigwa, can you consider using the recommended naming conventions for this PR title?

**Comment by Musigwa-zz** [comment]
I'm sorry for I accidentally closed the PR, I'm going to reraise it and I'll take into consideration all the modification suggestions.

**Pull request closed without merging**